inc/template-tags.php

```diff
@@ -18,8 +18,11 @@ function get_gtm_tag( string $container_id, array $data_layer = [], string $data
 	$data_layer_var = preg_replace( '/[^a-z0-9_\-]/i', '', $data_layer_var );
 
 	if ( ! empty( $data_layer ) ) {
-		$tag .= sprintf(
-			'<script>var %1$s = [ %2$s ];</script>',
+		$tag .= sprintf('
+			<script>
+			window.%1$s = window.%1$s || [];
+			window.%1$s.push([ %2$s ]);
+			</script>',
```

Comment:
```suggestion
		$tag .= sprintf(
			'<script>window.%1$s = window.%1$s || []; window.%1$s.push([ %2$s ]);</script>',
```
Output in the head might look a bit weirdly formatted. Not a big deal really, just the sort of nonsense I think about.

Reply:
Nonsense is what we do 😎

Reply:
Released 2.0.8